jaseci_core/jaseci/actions/standard/file.py

```diff
@@ -40,3 +42,38 @@ def dump_json(fn: str, obj, indent: int = None):
     """Standard built in for dumping json to file from dictionary"""
     with open(fn, 'w') as file:
         json.dump(obj, file, indent=indent)
+
+
+@jaseci_action()
+def build_form_data_from_b64(files: list):
```

Comment:
Are `files` in this case  a list of bytestrings? If so we should probably call this parameter `byte_strs` or `byte_strings`

Or perhaps it seems like to goal of this lib call would be to get a list of decoded filenames from b64 encoded strings, if so perhaps we can keep it as `files` but change the call from `build_form_data_from_b64` to `get_files_from_b64`

Would love to hop on a quick zoom to discuss.

Reply:
Actually, I'm planning to remove this and replace it with **base64_to_bytesio** (just decoding base64 and convert it to BytesIo)

The reason for adding **build_form_data_from_b64** is we need to upload file to a third party service. Their API uses multipart/form-data. I'm not fully sure what other approach to this other than using requests.post with array of tuples for files.

**build_form_data_from_b64** accepts json array
```
[
{
"name": "fileName.ext",
"base64": "base64=="
}
]
```
and will return array of tuples used as form-data 
```
[
('file', (fileName, Bytesio))
]
```

If I can declare tuples on jac file as same with python I think I can use the **base64_to_bytesio** instead